wntr/network/model.py

```diff
@@ -9,13 +9,13 @@
     WaterNetworkModel
     PatternRegistry
     CurveRegistry
-    SourceRegistry
```

Comment:
We need to keep the SourceRegistry. This links sources to patterns and nodes, and prevents the deletion of a node that is used by a source until the source is removed.  I tested this out with Net2 and it works as expected.

Reply:
Added back in, so hopefully the tests work.